standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClientPreCatalog.java

```diff
@@ -569,9 +553,16 @@ private void open() throws MetaException {
                   + "Continuing without it.", e);
             }
           }
+        } catch (TTransportException e) {
+          tte = e;
+          if (LOG.isDebugEnabled()) {
+            LOG.warn("Failed to connect to the MetaStore Server...", e);
```

Comment:
If the isDebugEnabled should not be Log.debug() instead of Log.Warn?

Reply:
`isDebugEnabled` is used to determine if we need print the full exception, I think it's OK to remove it because we'll print the full exception if the client was not connencted finally,